### ISSUE TITLE
Stuck focus between applications after using any mouse buttons

### ISSUE THREAD

**Issue by rmlebioda** [i3/i3#4758]
## I'm submitting a…

<pre>
[x] Bug
[ ] Feature Request
[ ] Documentation Request
[ ] Other (Please describe in detail)
</pre>

## Current Behavior
After pressing any mouse key (fe. left click), i3 focus is stuck on current window and moving mouse cursor to other windows, does not change focus and I can only interact with the first window I clicked (and interaction is also limited, see end of this issue for some examples, like dolphin, KDE).

## Expected Behavior
After pressing any mouse key on active window, I expected i3 focus to not be limited to window, I originally clicked on even after moving cursor to another window.

## Reproduction Instructions
1. Have two windows opened side by side (or at 2 screens).
2. Hover on one window and left mouse click on it (or scroll).
3. Move cursor to another window - focus is still stuck on previous window and any interactions with another window does not work.

## Environment

Output of `i3 --moreversion 2>&-`:
<pre>
i3 version: 
Binary i3 version:  4.20.1 (2021-11-03) © 2009 Michael Stapelberg and contributors
(Getting version from running i3, press ctrl-c to abort…)
[KRunning i3 version: 4.20.1 (2021-11-03) (pid 2049)
Loaded i3 config:
  /home/nuro/.config/i3/config (main) (last modified: Wed 29 Dec 2021 07:38:44 AM CET, 7998 seconds ago)

The i3 binary you just called: /usr/bin/i3
The i3 binary you are running: /usr/bin/i3
</pre>

<details><summary>Config file</summary>
https://pastebin.com/ruM8ihLq
</details>

<pre>
Logfile URL: https://logs.i3wm.org/logs/6325461206433792.bz2
</pre>

I had one screen with two windows opened: firefox and konsole. I restarted i3 with logging at 09:57:58.
At 09:58:00 I clicked on konsole application and I tried to move cursor to firefox - back and forth, focus is stuck on konsole.
At around 09:58:05 I right clicked in konsole to bring context menu, left clicked in konsole (outside context menu, to close it) and moved cursor to firefox - focus changed properly, working as expected.

<pre>
- Linux Distribution & Version: Manjaro, KDE Plasma Version: 5.23.4, Kernel: 5.15.7-1-MANJARO (64-bit), graphic platform: X11
- Are you using a compositor (e.g., xcompmgr or compton): compton
</pre>

Some other info:
Mouse: Logitech G502, Logitech G600 (tested both).

When focus is stuck at one application, I can do 2 things to solve it:
- if possible, right click and open context menu, after closing it, changing focus should be working fine (it kinda feels like on mouse button click, i3 is always waiting for context menu to close even if it did not open)
- restart i3 with $mod+shift+r

I tried to rename my config and load default one, but same issue there. It happened today suddenly, it never worked like that earlier. Yesterday I updated system with testing branch. I switched to unstable, then stable branch and downgraded all my packages, but it still behaves as described in this issue.

Some applications like dolphin does not let me ever left click on anything, only scrolling works. Any interaction with KDE taskbar/notifications area is also not working at all.
Overall, this issue is very annoying and makes i3 completely unusable, forcing to pretty much restart i3 every single time you want to change focus between windows.

**Comment by orestisfl** [comment]
Can you try without Compton?

**Comment by rmlebioda** [comment]
Thanks for reply, but weirdly enough I cannot reproduce this issue anymore. I disabled compton, restarted PC and I can no longer reproduce it (and yes, I restarted my pc twice before I posted this issue). I enabled compton again in i3 config, restarted i3 with shortcut and it still works fine (I also enabled compton from console). There is probably something more that needs to be done to reproduce it, but I cannot think of anything else at this point. If I won't get back to this issue in next few days, then I would probably close this issue. I need to use my PC for a longer period of time to provide more feedback.
